src/psyclone/nemo.py

```diff
@@ -53,23 +54,6 @@
 NEMO_SCHEDULE_COLOUR_MAP = copy.deepcopy(_BASE_CMAP)
 NEMO_SCHEDULE_COLOUR_MAP["CodeBlock"] = "red"
 
-# The valid types of loop and associated loop variable and bounds
```

Comment:
Please could you put the comments that used to be here in the Config file instead?

Reply:
Done